### PR TITLE
[Docs update]: Add note to README for protocol relative URL

### PR DESCRIPTION
**Pull request by sumitramanga**
This PR is to clearly define the relative protocol options available so that it's known that the CDN fetch may not work for projects with no server running (local hosting)

**Screenshot** 📸
Potentially how it could look (incomplete - markdown supports list views) -

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ No installation via a package manager is required. Just add the following snippe
 
 The above snippet will fetch the Raygun4JS script from our CDN asynchronously, so it doesn't block other scripts from being loaded. It will also catch errors that are thrown while the page is loading, and send them when the script is ready.
 
+
+**Note:** If you encounter a situation where no events are appearing within Raygun, you may need to hard code the protocol so that the CDN matches your hosting environment. This could look like one of the following -
```
Comment (Hamish-taylor) [inline README.md:28]:
```suggestion
**Note:** If you encounter a situation where no events are appearing within Raygun, you may need to hard code the URL protocol inside of the CDN loading script so that it matches your hosting environment. This could look like one of the following -
```

**Review by Hamish-taylor — CHANGES_REQUESTED** [review]
Looks good! just a few wording suggestions. Feel free to ignore them if they don't make sense 🙂

File: README.md (hunk 1 above)
Comment (sumitramanga) [reply README.md]: I think this has become too wordy but I've added URL :)

**Review by Hamish-taylor — APPROVED** [review]
LGTM! 🙌
